.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+# Description
+
+Addresses # (issue)
+
+Please include a summary of the change and which issue is fixed. Please also
+include relevant motivation and context. List any dependencies that are required
+for this change.  Please see ``CONTRIBUTING.md`` for more guidelines.
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality
+      to not work as expected)
+- This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide
+instructions so we can reproduce. Please also list any relevant details for
+your test configuration
+
+- Test A
+- Test B
+
+**Test Configuration**:
+* Operating system
+* Version number
+* Any details about your local setup that are relevant
+
+# Checklist:
+
+- [ ] Make sure you are merging into the ``develop`` (not ``master``) branch
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes
```

Comment:
Is there an Authors list or Zenodo authors json?  That could go on the checklist, for people to add themselves there.

Reply:
There is not, but it would make sense to add that as part of this pull

Reply:
Added the zenodo file based on the main pysat version, then updated based on the contributors here.